_commands/toys/cowsay.md

```diff
@@ -3,22 +3,62 @@
 
 Cowsay
 ---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+'cowsay' is a program that shows a ASCII picture of a cow saying a message inside the terminal.
 
-<!-- minimal example -->
-~~~ bash
-*short* example of command usage and output
+To use 'cowsay', simply type:
+
+~~~bash
+$ cowsay "your message"
 ~~~
 
-<!--more-->
```

Comment:
This `<!--more-->` comment needs to stay here so that extra information isn't displayed on the `/reference` page.